### PR TITLE
fix(whiteboard): window-manager repeat mount when whiteboard unMount

### PR DESCRIPTION
**Pull request by Cheerego7**

### CODE REVIEW

File: desktop/renderer-app/src/components/Whiteboard.tsx [hunk 1]
```diff
@@ -11,18 +11,21 @@ import { WhiteboardStore } from "../stores/whiteboard-store";
...
 
 export const Whiteboard = observer<WhiteboardProps>(function Whiteboard({ whiteboardStore }) {
     const { room } = whiteboardStore;
+    const isUnMountedRef = useIsUnMounted();
 
     const [whiteboardEl, setWhiteboardEl] = useState<HTMLElement | null>(null);
     const [collectorEl, setCollectorEl] = useState<HTMLElement | null>(null);
 
     useEffect(() => {
+        let isLoaded = false;
```
Comment (BlackHole1) [inline desktop/renderer-app/src/components/Whiteboard.tsx]: remove

File: desktop/renderer-app/src/components/Whiteboard.tsx [hunk 2]
```diff
@@ -40,12 +43,23 @@ export const Whiteboard = observer<WhiteboardProps>(function Whiteboard({ whiteb
...
+                    whiteboardStore.onMainViewModeChange();
+                    whiteboardStore.onWindowManagerBoxStateChange();
+                }
             }
         };
 
         void mountWindowManager();
+
+        return () => {
+            if (isLoaded) {
```
Comment (BlackHole1) [inline desktop/renderer-app/src/components/Whiteboard.tsx]: isUnMountedRef.current

**Review by BlackHole1 — CHANGES_REQUESTED** [review]
@crimx

File: desktop/renderer-app/src/components/Whiteboard.tsx (hunk 1 above)
Comment (crimx) [reply desktop/renderer-app/src/components/Whiteboard.tsx]: Ok

**Review by BlackHole1 — APPROVED** [review]

**Review by crimx — APPROVED** [review]

**Pull request merged**